index.js

```diff
@@ -25,7 +28,6 @@ class HomeBridgeTasmotaAirconHTTP {
     this.superagent = superagent; // TODO: Is superagent needed?
 
     this.fanSteps = config.fanSteps || 5; // Used to translate {0..100} to {1..fanSteps}
-    this.state = this._initialState(config);
```

Comment:
Removing this will break many of the tests. I think this would be better:
```
if (api.user) storage.initSync({'dir': path.join(api.user.storagePath(), 'HomeBridgeTasmotaAirconHTTP')});
this.state = storage.getItemSync(this.name) || this._initialState(config);
```

Also, I'm not sure about the "HomeBridgeTasmotaAirconHTTP" name. Can you give me a reference to where CamelCase is used?

Reply:
Another thing I just thought about: You *have* to call _initialState() first, since the storage might not have a complete view of the required state. Meaning that after `this.state` is initialized, then you can overwrite any values with data from storage inside the `if (...)` below.